.travis.yml

```diff
@@ -1,48 +1,41 @@
 language: python
-
 sudo: required
-
 python:
-    - "2.7"
-    - "3.4"
-    - "3.5"
-    - "3.6"
-    - "pypy"
-    - "pypy3"
-    - "nightly"
-
+- '2.7'
+- '3.4'
+- '3.5'
+- '3.6'
+- pypy
+- pypy3
+- nightly
 matrix:
-    allow_failures:
-      - python: "pypy3"
-
+  allow_failures:
+  - python: pypy3
 cache:
   directories:
-    - $HOME/.cache/pip
-
+  - "$HOME/.cache/pip"
 before_install:
-    - docker pull progrium/consul
-    - docker run -d -p 8400:8400 -p 8500:8500 -p 8600:53/udp -h node1 progrium/consul -server -bootstrap
-
+- docker pull progrium/consul
+- docker run -d -p 8400:8400 -p 8500:8500 -p 8600:53/udp -h node1 progrium/consul
+  -server -bootstrap
 install:
-    - make requirements
-    - pip install coveralls
-
+- make requirements
+- pip install coveralls
 script:
-    - if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then make isort-check; fi
-    - make test
-
+- if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then make isort-check; fi
+- make test
 after_success:
-    - coveralls
-
+- coveralls
 services:
-    - redis
-    - docker
-
+- redis
+- docker
 deploy:
   provider: pypi
   user: drgarcia1986
-  distributions: "sdist bdist_wheel"
+  distributions: sdist bdist_wheel
```

Comment:
if you drop quotations, travis only run `sdist`..

Reply:
I didn't it 😱 
I only ran `travis encrypt` command 🤔

Reply:
Are you sure about it? I can change without problems, I only want to understand

Reply:
We fix this issue on https://github.com/luizalabs/django-toolkit/blob/master/.travis.yml#L31

Reply:
Great!

Reply:
@jairhenrique done, could you check again?